modules/nodes-nodes-working-deleting-bare-metal.adoc

```diff
@@ -24,21 +24,25 @@ the following steps:
 $ oc adm cordon <node_name>
 ----
 
-. Drain all pods on your node:
+. Drain all pods on the node:
 +
 [source,terminal]
 ----
 $ oc adm drain <node_name> --force=true
 ----
 
-. Delete your node from the cluster:
+. If the node is unresponsive, it might be running a workload that writes to shared storage. To avoid data corruption, power down the physical hardware.
```

Comment:
One way to parse this sentence is that writing to shared storage might be the cause of the node being unresponsive.  Slight wording tweak to avoid that.  

```suggestion
. This step may fail if the node is offline or unresponsive.  Even if the node is not responding, it might still be running a workload that writes to shared storage. To avoid data corruption, power down the physical hardware.
```

Reply:
Thanks, @beekhof! I incorporated your suggestion into step 2: [preview](https://deploy-preview-32504--osdocs.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-deleting-bare-metal_nodes-nodes-working)